Makefile

```diff
@@ -35,6 +35,14 @@ clean:
 check-style: golangci-lint
 	@echo Checking for style guide compliance
 
+.PHONY: check-deps
+check-deps:
+	$(GO) mod tidy -v
+	@if [ -n "$$(command git --no-pager diff --exit-code go.mod go.sum)" ]; then \
```

Comment:
Is there a need to execute via command?

Reply:
will add this a make rule similar we did for others, this is a good check we have in other repos, and was running here as well, just replicating the same ci again